stl/inc/flat_set

```diff
@@ -621,32 +628,30 @@ private:
 
     void _Erase_dupes_if_not_multi() {
         if constexpr (!_Multi) {
-            const auto _Equal_to = [this](const _Kty& _Lhs, const _Kty& _Rhs) {
+            const auto _Equivalent = [this](const _Kty& _Lhs, const _Kty& _Rhs) {
                 return !_Compare(_Lhs, _Rhs) && !_Compare(_Rhs, _Lhs);
             };
-            const iterator _End     = end();
-            const iterator _New_end = _STD unique(begin(), _End, _Equal_to);
-            _Get_cont().erase(_New_end, _End);
+            const iterator _End = end();
+            _Mycont.erase(_STD unique(begin(), _End, _Equivalent), _End);
         }
     }
 
     template <bool _Presorted>
     void _Restore_invariants_after_insert(const size_type _Old_size) {
-        auto _Comp              = _Get_comp_v();
         const iterator _Begin   = begin();
         const iterator _Old_end = _Begin + static_cast<difference_type>(_Old_size);
-        const iterator _New_end = end();

```

Comment:
The two `_New_end` were sharing different meanings in `_Erase_dupes_if_not_multi` and here. Renamed to common `_End` here to match with other methods.